client/src/components/pages/Settings.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classes from './../../styles/Settings.module.css';
+
+const Set = () => {
+    return (
+        <main>
+            <div className={classes.settings_container}>
```

Comment:
в js мы используем camelCase даже для css модулей
```suggestion
            <div className={classes.settingsContainer}>
```